### ISSUE TITLE
[Question]  Mysql Monitor Template fields And Monitor Rule

### ISSUE THREAD

**Issue by QBH-insist** [apache/hertzbeat#2289]
### Question

"Version"： `v1.6`

1. The Chinese explanation for the 'Connections' field in the MySQL monitoring template seems to be incorrect

![image](https://github.com/user-attachments/assets/812b6efe-28f7-4f40-9e9b-8760513a5e8c)

https://dev.mysql.com/doc/refman/5.7/en/server-status-variables.html
![image](https://github.com/user-attachments/assets/efcd23ab-55d6-47ff-bfdc-47f68eafc183)

2. Then, I want to set the rule '${threads_connected} >=({max_connections}*0.9)'. Is this rule available and feasible, considering that these two data points are located on different tabs?

**Comment by tomsun28** [comment]
can you help sumbit a pr to fix it?

**Comment by Yanshuming1** [comment]
Second, are you trying to set threshold rules? General calculates calculates should not write '>' matches, and they must be under the same name if you want to set display

**Comment by QBH-insist** [comment]
> can you help sumbit a pr to fix it?

https://github.com/apache/hertzbeat/pull/2294

**Comment by QBH-insist** [comment]
> Second, are you trying to set threshold rules? General calculates calculates should not write '>' matches, and they must be under the same name if you want to set display
Yes, I was wondering if it's possible to compare ${BasicInfo#MaxConnections} with ${ThreadInfo#ThreadsConnected} within a mysql threshold rule

**Comment by Yanshuming1** [comment]
> > Second, are you trying to set threshold rules? General calculates calculates should not write '>' matches, and they must be under the same name if you want to set display
> 
> Yes, I was wondering if it's possible to compare ${BasicInfo#MaxConnections} with ${ThreadInfo#ThreadsConnected} within a mysql threshold rule

No, he collects data based on name each time, not all name data at one time, and the calculation of alarms can only be calculated based on the fields under name。
@tomsun28  Am I explaining this correctly？

**Comment by tomsun28** [comment]
hi, yes, not support now. But this is a common requirement. Let's see how to implement it later.
